### PR TITLE
hardening/236

### PR DESCRIPTION
**Pull request by frbattid**
Fixes issue https://github.com/telefonicaid/fiware-connectors/issues/236

### CODE REVIEW

File: flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java [hunk 1]
```diff
@@ -61,18 +61,18 @@
 public abstract class OrionSink extends AbstractSink implements Configurable {
 
     private Logger logger;
-
```
Comment (frbattid) [inline flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java]: Do not know why this deletions and insertions... maybe something related with environment used by @crbrox , who created the branch.

Comment (fgalan) [inline flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java]: NTC

File: flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java [hunk 2]
```diff
@@ -137,22 +137,35 @@ public Status process() throws EventDeliveryException {
...
-                    event.getHeaders().put(Constants.HEADER_TTL, newTTL);
+                try {
+                    ttl = Integer.parseInt(ttlStr);
+                } catch (NumberFormatException nfe) {
+                    ttl = 0;
+                    logger.error("Invalid TTL value (id=" + event.hashCode() + ", ttl=" + ttlStr
+                          +  ", " + nfe.getMessage() + ")");
+                } // try catch
+                
+                if (ttl == -1) {
```
Comment (fgalan) [inline flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java]: Does documentation specifies that ttl = -1 means "forever" ?

Comment (frbattid) [inline flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java]: Fixed in https://github.com/telefonicaid/fiware-connectors/commit/4f731fe50691d4b39c4d173d6a650c37fe72f7ed

**Comment by fgalan** [comment]
LGTM

**Pull request merged**